tests/flow/test_mix_labels.py

```diff
@@ -0,0 +1,119 @@
+import os
+import sys
+import unittest
+from redisgraph import Graph, Node, Edge
+
+# import redis
+from .disposableredis import DisposableRedis
+
+from base import FlowTestsBase
+
+redis_graph = None
+male = ["Roi", "Alon", "Omri"]
+female = ["Hila", "Lucy"]
+
+def redis():
+    return DisposableRedis(loadmodule=os.path.dirname(os.path.abspath(__file__)) + '/../../src/redisgraph.so')
+
+class GraphMixLabelsFlowTest(FlowTestsBase):
+    @classmethod
+    def setUpClass(cls):
+        print "GraphMixLabelsFlowTest"
+        global redis_graph
+        cls.r = redis()
+        cls.r.start()
+        redis_con = cls.r.client()
+        redis_graph = Graph("G", redis_con)
+
+        # cls.r = redis.Redis()
+        # redis_graph = Graph("G", cls.r)
+
+        cls.populate_graph()
+
+    @classmethod
+    def tearDownClass(cls):
+        cls.r.stop()
+        # pass
+
+    @classmethod
+    def populate_graph(cls):
+        redis_graph
```

Comment:
This line can be safely deleted here and everywhere else in this file (start of each `test_` function), as variables are assumed to be global unless they are explicitly assigned within the function.